paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/ResourceRepository.kt

```diff
@@ -12,8 +12,12 @@ internal class ResourceRepository @Inject internal constructor(
     internal val bankRepository: BankRepository,
     internal val addressRepository: AddressFieldElementRepository
 ) {
+    private var initialized = false
```

Comment:
now that this is initialized in the top graph(`DefaultFlowController`/`PaymentSheetLauncher`), I think we can just call `bankRepository.init()` and `addressRepository.init()` in `ResourceRepository`'s native `init` block and avoid the explicit `init()` method call

Reply:
Will the dagger construction happen on a background thread?  Just want to make sure that init happens in the background.

Reply:
Yes, it happens in a dispatcher and won't block main thread